python/dgl/distributed/dist_graph.py

```diff
@@ -319,7 +320,7 @@ class DistGraph:
     def __init__(self, ip_config, graph_name):
         connect_to_server(ip_config=ip_config)
         self._client = KVClient(ip_config)
-
+        graph_name = "{}_{}".format(graph_name, self._client.machine_id)
```

Comment:
I changed graph_name to graph_name+partition_id, to avoid conflict name when testing on the same machine

Reply:
the problem is how the client knows which partition it should map?